chart/openfaas/values.yaml

```diff
@@ -174,6 +174,17 @@ operator:
     requests:
       memory: "120Mi"
       cpu: "50m"
+     
+# The values for jetstreamQueueWorker are merged with those under
+# the "queueWorkerPro" and "queueWorker" section
+#  
+# Enable automatically when nats.queueMode is set to "jetstream"
+jetstreamQueueWorker:
+  image: ghcr.io/openfaasltd/jetstream-queue-worker:0.1.0-rc2
```

Comment:
Please update to `0.1.0-rc3` when this is published, I've started a build, which you can monitor on the actions tab of the repo.